src/script/Environment.ts

```diff
@@ -83,6 +84,9 @@ export const IS_SELF_HOSTED = window.wire.env.IS_SELF_HOSTED;
 export const NEW_PASSWORD_MINIMUM_LENGTH = window.wire.env.NEW_PASSWORD_MINIMUM_LENGTH || DEFAULT_PASSWORD_MIN_LENGTH;
 export const VERSION = window.wire.env.VERSION;
 
+export const ENABLE_DEV_BACKEND_API = window.wire.env.FEATURE.ENABLE_DEV_BACKEND_API;
+export const SUPPORTED_API_RANGE = [1, window.wire.env.FEATURE.ENABLE_DEV_BACKEND_API ? Infinity : 10];
```

Comment:
Shouldn't we align with the current webapp RC?
```suggestion
export const SUPPORTED_API_RANGE = [1, window.wire.env.FEATURE.ENABLE_DEV_BACKEND_API ? Infinity : 11];
```